README.md

```diff
@@ -49,55 +50,65 @@ pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
 ## Quickstart
 
 ```python
-import keras_cv
 import tensorflow as tf
 from tensorflow import keras
+from keras_cv.layers import RandomFlip, RandAugment, CutMix, Mixup
```

Comment:
maybe from keras_cv import layers as cv_layers?  Typically we don't really like to prescribe Symbol imports as it hurts trace-ability a bit.